### ISSUE TITLE
Update iOS sdk FirebaseRemoteConfig cause crash on cpp sdk remote_config_ios.mm

### ISSUE THREAD

**Issue by bolso80** [firebase/firebase-cpp-sdk#202]
### [REQUIRED] Please fill in the following fields:

  * Pre-built SDK from the [website](https://firebase.google.com/download/cpp) 
  * Firebase C++ SDK version: 6.16.1
  * Problematic Firebase Component: RemoteConfig 
  * Other Firebase Components in use: Analytics, Crashlytics 
  * Platform you are using the C++ SDK on: Mac
  * Platform you are targeting: iOS

### [REQUIRED] Please describe the issue here:

When you update to last versions of firebase cpp sdk and firebase ios sdk, has a crash on call to ActivateFetched() method.

[Line method crash](https://github.com/firebase/firebase-cpp-sdk/blob/f0a9d5b65e3a014aee347ea9bb3b664cdce2ae55/remote_config/src/ios/remote_config_ios.mm#L350)

[Commit on firebase ios sdk where deleted method used on cpp sdk](https://github.com/firebase/firebase-ios-sdk/commit/f56afe9ba124c2de0561a0ea2bc564405846b393)

#### Steps to reproduce:

Update to last versions of firebase cpp sdk and firebase ios sdk.
Remote Config Initiliaize, fetch and call to ActivateFetched().

Have you been able to reproduce this issue with just the [Firebase C++ quickstarts](https://github.com/firebase/quickstart-cpp) ?
Yes
What's the issue repro rate? (eg 100%, 1/5 etc)
100%
What happened? How can we make the problem occur?
> [FIRRemoteConfig activateWithCompletionHandler:]: unrecognized selector sent to instance 0x280209540

**Comment by DellaBitta** [comment]
Hi @bolso80,

When you say that you've upgraded both the Firebase C++ SDK *and* the Firebase iOS SDK, do you mean that you're attempting to download and use the Firebase C++ 6.16.1  as is, or are you attempting to use Firebase C++ 6.16.1 with Firebase iOS SDK 7.0 or 7.1? 

I ask because the Firebase C++ SDK v6.16.1 was built and tested against Firebase iOS SDK version 6.32.2.  We're currently working on a release that will be built against Firebase iOS SDK 7.0.

Thanks!

**Comment by bolso80** [comment]
Hi @DellaBitta 

Well, yes, I attempting to use Firebase C++ 6.16.1 with Firebase iOS SDK 7.0 and crashed on some methods of Remote Configuration. 
I downloded a previous version of iOS SDK ( I think was 6.32.2) when I saw some methods how was deleted and provoked the crash. (I dont saw on firebase docs the compatibility of C++ sdk with iOS SDK versions)

Then I have to wait to the next release how working against iOS SDK 7.0.

Thanks
